src/components/developers/developers.styles.js

```diff
@@ -0,0 +1,99 @@
+import { createUseStyles } from 'react-jss'
+
+const useDevelopersStyles = createUseStyles(theme => ({
+  section: {
+    padding: theme.spacing(2),
+    [theme.breakpoints.sm]: {
+      padding: '96px 72px'
```

Comment:
theme.spacing for padding?

Reply:
Added (have it in another branch, but I changed it here anyway).